.github/workflows/ci_tests.yml

```
@@ -31,12 +31,15 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        platform: [ubuntu-latest]
-        compiler:
-          - cpp: g++
+        platform:
+          - description: "ubuntu gcc"
```

```suggestion
          - description: "Ubuntu GCC"
```
if it's in description?

---

Adopted.